.github/* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12

```diff
@@ -0,0 +1 @@
+CODEOWNERS
```

Comment:
I think the name of the file should be CODEOWNERS and this should be the content:
`@saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12`